puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman_infra.groovy

```diff
@@ -10,13 +10,13 @@ def list_files(glob = '') {
     sh(script: "ls -1 ${glob}", returnStdout: true).trim().split()
 }
 
-def set_job_build_description(job_name) {
+def set_job_build_description(job_name, status) {
     def build_description = ""
     def file_name = "jobs/${job_name}"
 
     if (fileExists(file_name)) {
        link = readFile(file_name)
-       build_description += "<a href=\"${link}\">${job_name}</a><br/>"
+       build_description += "<a href=\"${link}\">${job_name}</a> (${status})<br/>"
```

Comment:
This would put it on the same line right at the end of the link. I think it might look better to do a `<br/>Status: ${status}<br/>` type of formatting.

Reply:
The intent was to keep this short (`(unknown)`, `(passed)` and `(failed)`) so you knew what it was relating to. If it's a new line and you have multiple builds then it can become unclear if it belongs to the one above or below. Given we have 3 builds in the Deb builds, that's a real risk with the middle one.